apps/docs/src/app/core/component-docs/table/table-docs.component.html

```diff
@@ -164,6 +164,23 @@
 
 <separator></separator>
 
+<fd-docs-section-title [id]="'with-row-unnavigatable'" [componentName]="'table'">
+    Table with Non-navigatable Row
+</fd-docs-section-title>
+<description>
+    A row can be non-navigatable by setting the property <code>[navigatable]="false"</code>, for both regular and Pop In modes.<br><br>
+    When an entire row is used for navigation, the navigation indicator will hide when the row becomes non-navigatable.<br>
+    To hide indicator, leave the <code>glyph</code> property empty i.e. <code>[glyph]=""</code>, and set <code>[navigation]="false"</code> for the <code>fd-table-icon</code> directive.<br><br>
```

Comment:
I didn't see ` [glyph]=""` anywhere in the example code.

Reply:
In the example code, there is a ```getGlyph``` method - [Link](https://github.com/SAP/fundamental-ngx/pull/5526/files#diff-bc3e0122cdaefee2617a9bb755c32219cf7c05c957383b1e44c30577ce604407R112). So when the row is not navigatable, it's equivalent to```[glyph]=""```. I see this could be better worded.